examples/vpc-with-dns/variables.tf

```diff
@@ -0,0 +1,56 @@
+variable "ibmcloud_api_key" {
+  description = "APIkey that's associated with the account to provision resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  description = "The region to which to deploy the VPC"
+  type        = string
+  default     = "us-south"
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources"
+  type        = string
+  default     = "dns"
+}
+
+variable "name" {
+  description = "The name of the vpc"
+  type        = string
+  default     = "sz-vpc"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "resource_tags" {
+  description = "List of Tags for the resource created"
+  type        = list(string)
+  default     = null
+}
+
+variable "dns_records" {
```

Comment:
The standalone example would not run without the `dns_records`.

Reply:
Yes right. Provided the default value. Thank you.